### PR TITLE
Feature/alcs 646

### PR DESCRIPTION
**Pull request by mhuseinov**
- release draft decision
- link reconsiderations and modifications to subsequent decisions
- added createAt date column for isFirst decision calculation
- unit tests

### CODE REVIEW

File: alcs-frontend/src/app/services/application/application-detail.service.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { BehaviorSubject } from 'rxjs';
+import { BehaviorSubject, MonoTypeOperatorFunction } from 'rxjs';
```
Comment (dhaselhan) [inline alcs-frontend/src/app/services/application/application-detail.service.ts:2]: Accidental import?

File: portal-frontend/src/app/features/home/application-list/application-list.component.ts [hunk 2]
```diff
@@ -33,6 +33,8 @@ export class ApplicationListComponent implements OnInit {
     [APPLICATION_STATUS.INCOMPLETE, 'incomplete'],
     [APPLICATION_STATUS.WRONG_GOV, 'wrong-government'],
     [APPLICATION_STATUS.CANCELLED, 'cancelled'],
+    [APPLICATION_STATUS.ALCDECISION, 'submitted-to-alc'],
+    [APPLICATION_STATUS.CEODECISION, 'submitted-to-alc'],
```
Comment (dhaselhan) [inline portal-frontend/src/app/features/home/application-list/application-list.component.ts:37]: I assume the TODO is to add the statuses later?

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

File: alcs-frontend/src/app/services/application/application-detail.service.ts (hunk 1 above)
Comment (mhuseinov) [reply alcs-frontend/src/app/services/application/application-detail.service.ts:2]: yep, thanks

File: portal-frontend/src/app/features/home/application-list/application-list.component.ts (hunk 2 above)
Comment (mhuseinov) [reply portal-frontend/src/app/features/home/application-list/application-list.component.ts:37]: good catch, I forgot to add custom colour for portal

**Pull request merged**